### PR TITLE
One client

### PR DESCRIPTION
**Pull request by ke-zhang-rd**
This branch using per db per test function instead of per db per example comparing to PR #3

### CODE REVIEW

File: amostra/tests/test_revert.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    for name in names[1:]:
+        s.name = name 
+
+    num = random.randint(0, n-2)
+
+    revert_target_cursor = client._db.samples_revisions.find({'revision': num, 'uuid': uuid})
+    s.revert(num)
+    target = next(revert_target_cursor)
+    for name, trait in s.traits().items():
+        if name is 'revision':
```
Comment (danielballan) [inline amostra/tests/test_revert.py]: Use `==` for string comparison. It's a fluke that `is` happens to work here. It won't always.

File: amostra/tests/test_revert.py [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+import random
+import uuid
+
+
+alphabet_list = ''
+for i in range(26):
+    alphabet_list = alphabet_list + chr(97 + i)
+
+
+@given(names = st.lists(st.text(alphabet=alphabet_list, min_size=1, max_size=4), min_size=3, max_size=4, unique=True))
```
Comment (danielballan) [inline amostra/tests/test_revert.py]: `string.ascii_lowercase` could be used in place of `alphabet_list`.

**Comment by danielballan** [comment]
Also, remember to run `flake8`. :-)

**Comment by ke-zhang-rd** [comment]
This should be ready to merge

**Comment by ke-zhang-rd** [comment]
This should be ready to merge.

**Pull request merged**